.github/workflows/authors.yml

```diff
@@ -0,0 +1,21 @@
+name: "authors update"
+on:
+  workflow_dispatch:
+
+jobs:
+  authors_update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0' # this is required to actually get all the authors
```

Comment:
```suggestion
          fetch-depth: 0 # this is required to actually get all the authors
```

Reply:
I don't think this is a nit worth retesting the script's functionality for. It works as is, and I'd prefer not to rock the boat with changes.

Reply:
There is no need to retest, unless you have a doubt, in which case I think a simple YAML validator is sufficient. Otherwise, removing unnecessary quotes might unload the action and make it a bit more readable. Personally, I don't see any problem with that.

Reply:
There is no need to retest, unless you have a doubt, in which case I think a simple YAML validator is sufficient. Otherwise, removing unnecessary quotes could make the workflow a bit more readable. Personally, I don't see any problem with that.